CHANGES.md

```diff
@@ -1,5 +1,94 @@
 # Changelog
 
+# 1.9.0 (core) / 0.25.0 (libraries)
+
+## Major changes since 1.8.0 (core) / 0.24.0 (libraries)
+
+### Automation
+
+- Declarative Automation, the system which enables setting per-asset `AutomationConditions` , is no longer experimental. We now recommend using this system in all cases where asset-centric orchestration is desired. A suite of built-in static constructors have been added for common usecases, such as `AutomationCondition.on_missing()` (which can fill in missing partitions of assets as soon as upstream data is available), and `AutomationCondition.all_deps_blocking_checks_passed()` (which can prevent materialization of assets until all upstream blocking checks have passed).
+- You can now assign `AutomationConditions` to asset checks, via the `automation_condition` parameter on `@asset_check` or `AssetCheckSpec`.
+- You can now assign `AutomationConditions` to observable source assets, via the `automation_condition` parameter on `@observable_source_asset`.
+- [experimental] You can now define custom subclasses of `AutomationCondition` to execute arbitrary Python code in the context of a broader expression. This allows you to compose built-in conditions with custom business logic.
+- The `target` arguments on schedules and sensors are now marked stable, allowing a stable way for schedules and sensors to target asset selections without needing to define a job.
+
+### Integrations
+
+- Introduced a slate of integrations with business intelligence (BI) tools, enabling dashboards, views, and reports to be represented in the Dagster asset graph.
+  - Power BI [docs]
+  - Looker [docs]
+  - Tableau [docs]
+  - Sigma [docs]
+- Pipes 1.9 changes: TODO
```

Comment:
@danielgafni @smackesey ping to update this

Reply:
or maybe @yuhan

Reply:
@OwenKephart do we want more here? like a short overview of Pipes? Or that's fine?

Reply:
@danielgafni A quick explanation of why we think Pipes is a superior interface (like 1-2 sentences) would be great